.github/workflows/prepare-for-staging-deploy.yml

```
@@ -139,30 +139,55 @@ jobs:
-              echo "No changes detected in staging environment. Version $TAG_NAME is already deployed. Skipping commit."
-              exit 0
-            fi
-
-            git commit -m "chore: upgrade staging env transformers to \"$TAG_NAME\""
-            git push -u origin $BRANCH_NAME
-            gh pr create --fill
+            echo "exists=false" >> $GITHUB_OUTPUT
           fi
+
+      - name: Update image tags in devops repo for staging
+        if: steps.check-branch.outputs.exists == 'false'
```

actionlint (1.7.11)

[error] property "check-branch" is not defined in object type {generate-token: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)